gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/AssignmentColumnHeaderPanel.java

```diff
@@ -58,7 +64,10 @@ public void onInitialize() {
 		final Assignment assignment = this.modelData.getObject();
 
 		// get user's role
-		final GbRole role = this.businessService.getUserRole();
+		final GbRole role = ((BasePage)getPage()).getRole();
```

Comment:
If not using the service this could just be passed through from the GradebookPage with a modified model/map.